src/schema/v1.json

```diff
@@ -3,34 +3,83 @@
  "$schema":"http://json-schema.org/draft-07/schema#",
  "$id":"http://github.com/coreos/coreos-assembler/blob/master/schema/v1.json",
  "type":"object",
- "title":"CoreOS v1 meta schema",
+ "title":"CoreOS Assember v1 base meta.json schema",
  "required": [
-   "buildid",
-   "coreos-assembler.basearch",
-   "coreos-assembler.build-timestamp",
-   "coreos-assembler.code-source",
-   "coreos-assembler.config-dirty",
-   "coreos-assembler.config-gitrev",
-   "coreos-assembler.container-config-git",
-   "coreos-assembler.container-image-git",
-   "coreos-assembler.image-config-checksum",
-   "coreos-assembler.image-genver",
-   "coreos-assembler.image-input-checksum",
-   "images",
-   "name",
-   "ostree-commit",
-   "ostree-content-bytes-written",
-   "ostree-content-checksum",
-   "ostree-n-cache-hits",
-   "ostree-n-content-total",
-   "ostree-n-content-written",
-   "ostree-n-metadata-total",
-   "ostree-n-metadata-written",
-   "ostree-timestamp",
-   "ostree-version",
-   "rpm-ostree-inputhash",
-   "summary"
-  ],
+     "name"
+ ],
+ "anyOf": [
+     {
+       "properties": {
+        "name": {
+            "const": "fedora-coreos"
+        },
+        "required": [
+           "buildid",
+           "coreos-assembler.basearch",
+           "coreos-assembler.build-timestamp",
+           "coreos-assembler.code-source",
+           "coreos-assembler.config-dirty",
+           "coreos-assembler.config-gitrev",
+           "coreos-assembler.container-config-git",
+           "coreos-assembler.container-image-git",
+           "coreos-assembler.image-config-checksum",
+           "coreos-assembler.image-genver",
+           "coreos-assembler.image-input-checksum",
+           "fedora-coreos.parent-commit",
+           "fedora-coreos.parent-version",
+           "images",
+           "name",
```

Comment:
Can't `name` be dropped from the `anyOf` of both `rhcos` & `fedora-coreos` because it's in the outer level `required` (line 8)?